### PR TITLE
Marios development

### PR DESCRIPTION
**Pull request by mariom3**
Add image and my name to webpage.

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
   margin-top: 20px;
 }
+
+/* Update to make image responsive with a max width*/
+#home_img {
+    text-align: center;
+}
+
+#home_img img{
+    max-width: 70%;
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css]: nit: space after img

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+/* Create flex box */
+
+#container {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css]: Please make the css properrties alphabetically ordered here and below. So,
display:
flex-direction:
justify-content:
etc

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
+
+#beginner-content {
+    text-align: center;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+/* Update to make image responsive with a max width*/
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css]: Please add 1 space before */

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,29 @@
-#content {
+/* Create flex box */
+
+#container {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css]: Please make the css properties alphabetically ordered here and in following rules. So,
display:
flex-direction:
justify-content:
etc

**Review by sbahaud — COMMENTED** [review]
Some minor comments. 
Could you share your url, so that I can take a look.

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (mariom3) [reply portfolio/src/main/webapp/style.css]: Done: Added space after img.

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (mariom3) [reply portfolio/src/main/webapp/style.css]: Done: Added space before the comment.

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (mariom3) [reply portfolio/src/main/webapp/style.css]: Done.

**Review by sbahaud — APPROVED** [review]
LGTM
Loved the photograph!

**Pull request merged**